tools/stylelint/theme-mixin-api.ts

```diff
@@ -147,7 +147,9 @@ const plugin = (isEnabled: boolean, _options: never, context: {fix: boolean}) =>
       }
 
       const expectedProperty = type === 'density' ? '$density-scale' : '$config';
-      const expectedValue = `mat-get-${type}-config($config-or-theme)`;
+      const expectedValue = type === 'typography' ?
+          `mat-private-typography-normalized-config(mat-get-typography-config($config-or-theme))` :
```

Comment:
Maybe instead of special-casing the `typography`, we should allow both variants?

Reply:
Yeah that's a good point, since I don't expect this all to necessarily go through in one PR